### PR TITLE
Fix Linux IIS credential issue.

### PR DESCRIPTION
**Pull request by frank117117**
Skip some tests for Linux IIS credential issue till we have a good solution.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/wcf?pullRequest=4761) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/wcf?pullRequest=4761) before we can accept your contribution.<br/><br/>:x: frank117117 [sign now](https://cla.dotnetfoundation.org/dotnet/wcf?pullRequest=4761)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/wcf?pullRequest=4761) it.</sub>

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs [hunk 1]
```diff
@@ -142,6 +142,19 @@ public static bool IsWindows()
             return OSID.AnyWindows.MatchesCurrent();
         }
 
+        public static bool IslinuxWithIIS()
+        {
+            if (IsWindows())
+            {
+                return true;
```
Comment (HongGit) [inline src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs:149]: should this be true or false?

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs [hunk 2]
```diff
@@ -142,6 +142,19 @@ public static bool IsWindows()
             return OSID.AnyWindows.MatchesCurrent();
         }
 
+        public static bool IslinuxWithIIS()
```
Comment (HongGit) [inline src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs:145]: Trying to understand what this is trying to determine?  Is this trying to determine whether this is Linux or not?  Or is trying to determine something else?  Is there a better name?

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs [hunk 3]
```diff
@@ -142,6 +142,19 @@ public static bool IsWindows()
...
+        {
+            if (IsWindows())
+            {
+                return true;
+            }
+            else if (!IsIISHosted())
```
Comment (HongGit) [inline src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs:151]: true or false?

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs (hunk 2 above)
Comment (frank117117) [reply src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs]: I have update suitable name to 'IsNotLinuxWithIIS'.

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs (hunk 3 above)
Comment (frank117117) [reply src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs:151]: I have update suitable name to 'IsNotLinuxWithIIS'.

File: src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs (hunk 1 above)
Comment (frank117117) [reply src/System.Private.ServiceModel/tests/Common/Infrastructure/ConditionalTestDetectors.cs:149]: I have update suitable name to 'IsNotLinuxWithIIS'.

**Review by HongGit — APPROVED** [review]

**Pull request merged**